meson.build

```
@@ -12,6 +12,25 @@ gresource = gnome.compile_resources(
     source_dir: 'data'
 )
 
+add_project_arguments('-DLIBICAL_GLIB_UNSTABLE_API=1', language: 'c')
+
+libecal_dep = dependency('libecal-2.0')
+
+tasks_deps = [
+    dependency('glib-2.0'),
+    dependency('gobject-2.0'),
+    dependency('granite', version: '>=0.5'),
+    dependency('gtk+-3.0'),
+    libecal_dep,
+    dependency('libedataserver-1.2'),
+    dependency('libhandy-1', version: '>=0.90.0'),
+    dependency('libical-glib')
+]
+
+if libecal_dep.version().version_compare('<=3.38.1')
```

```suggestion
if libecal_dep.version().version_compare('<=3.39')
```
there won't be backports of the these until next EDS release